lib/APMSpan.js

```diff
@@ -216,6 +219,19 @@ class APMSpan {
     isFinished() {
         return this[FINISHED];
     }
+
+    /**
+     * Deprecated
+     */
+    logEvent() {
+        console.log('Deprecated call to logEvent(eventName, payload)');
+    }
+
+    setBaggageItem() {
```

Comment:
Yes it should have, however, eslint would complain about unused parameters so it would not compile. Here is the jira https://issues.jboss.org/browse/HWKAPM-747 for baggage support.